relayer/relays/beefy/beefy-ethereum-listener.go

```diff
@@ -203,63 +272,79 @@ func (li *BeefyEthereumListener) processHistoricalInitialVerificationSuccessfulE
 					"initial_verification_tx": event.Raw.TxHash.Hex(),
 					"complete_on_block":       event.Raw.BlockNumber + li.blockWaitPeriod,
 				}
-				updateCmd := store.NewDatabaseCmd(item, store.Update, instructions)
-				li.dbMessages <- updateCmd
 
-				itemFoundInDatabase = true
+				select {
+				case <-ctx.Done():
+					return ctx.Err()
+				case li.dbMessages <- store.NewDatabaseCmd(item, store.Update, instructions):
+				}
+
+				found = true
 				break
 			}
 		}
-		if !itemFoundInDatabase {
+		if !found {
 			// Don't have an existing item to update, therefore we won't be able to build the completion tx
-			log.Error("BEEFY justification data not found in database for InitialVerificationSuccessful event. Ignoring event.")
+			return fmt.Errorf("item not found in database for InitialVerificationSuccessful event")
```

Comment:
i think we just want to ignore if there is nothing found, so maybe should not return an error?

Reply:
Yeah, OK this one I understand better now. Will remove the error check here.